src/packages/cli/utils/url-message.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+export default function urlMessage(): void {
+  console.log(`
+  You're using a URL in your database config (config/database.js).
+
+  In that case, Lux assumes you don't need to create or drop your database.
+  If you'd like to create or drop a database, set up your database config\
+ without the url.
+
+  For guidance, see:
+  https://github.com/postlight/lux/blob/master/src/packages/database/\
+interfaces.js#L17
```

Comment:
At minimum one thing I should probably add is that they could be using a url in the database config or via the DATABASE_URL environment variable.